### PR TITLE
Add support for HPPA

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ the docker images used to build the various GCC cross-compilers used on the site
 - check it builds correctly:
   `./ct-ng build`
 - copy the config in this repository in `build/latest` following the naming convention.
+- build the container as a one-off with `docker build -t gcc-cross .`
```
Comment (dkm) [inline README.md:15]: Oh, of course. Sorry I forgot that line :sweat_smile:

File: local_build.sh [hunk 2]
```diff
@@ -10,21 +10,27 @@ RET=0
 
 mkdir -p $OUTDIR
 
+DOCKER=docker
+if ! type -P ${DOCKER} 2>/dev/null;
+then
+    DOCKER=podman
+fi
+
```
Comment (dkm) [inline local_build.sh:18]: Oh nice. I'm still a newbie in all this. I never thought we could use podman directly :+1:

Comment (thesamesam) [reply local_build.sh:18]: I've got `alias docker=podman` in my bashrc :eyes:

File: README.md (hunk 1 above)
Comment (thesamesam) [reply README.md:15]: It took me a bit to realise!

**Pull request merged**